tests/test_ufonormalizer.py

```diff
@@ -765,6 +767,7 @@ def test_normalizeGLIF_lib_defined(self):
             writer.getText(),
             '<lib>\n\t<dict>\n'
             '\t\t<key>abc</key>\n\t\t<string></string>\n'
+            '\t\t<key>def</key>\n\t\t<data>\n\t\t</data>\n'
```

Comment:
shouldn't it be on the same line, like in the input you gave?

Reply:
yeah, perhaps in the `_plistData` method of XMLWriter, you could check if the value is empty and then use `simpleElement` instead of begin and endElement